robottelo/host_helpers/api_factory.py

```diff
@@ -39,6 +39,7 @@ def make_http_proxy(self, org, http_proxy_type):
                 name=gen_string('alpha', 15),
                 url=settings.http_proxy.un_auth_proxy_url,
                 organization=[org.id],
+                content_default_http_proxy=True,
```

Comment:
Why don't we take content_default_http_proxy as parameter with default value?

Reply:
For every proxy we create, we are setting it as default http proxy.

Reply:
yes, the fixture behaviour is still same, if we make it non-default then we need to configure the setting explicitely and that step we have removed it from this setup fixture

Reply:
Got it. Thanks.